src/win32/win32gliface.cpp

```diff
@@ -1141,7 +1141,7 @@ void Win32GLFrameBuffer::ReleaseResources ()
 
 void Win32GLFrameBuffer::SetVSync (bool vsync)
 {
-	if (vsyncfunc != NULL) vsyncfunc(vsync);
+	if (vsyncfunc != NULL) vsyncfunc(vsync ? 1 : 0);
```

Comment:
I know, but the argument to wglSwapInterval is not a boolean in the semantic meaning. For example, 2 is a valid value to this function. By passing the bool directly it gives the reader of the code the idea that it takes a true/false argument when it in fact does not.

I originally made the change because I wasn't 100% sure whether a C++ true value always maps to 1 (seems it does), but nevertheless I kept the change because I find it semantically wrong.